server/src/test/java/io/javaclasses/fileHub/storage/TokenStorage/LoggedInUsersStorageTest.java

```diff
@@ -0,0 +1,71 @@
+package io.javaclasses.fileHub.storage.TokenStorage;
+
+import com.google.common.testing.NullPointerTester;
+import io.javaclasses.filehub.storage.tokenStorage.Token;
+import io.javaclasses.filehub.storage.tokenStorage.LoggedIdUserRecord;
+import io.javaclasses.filehub.storage.tokenStorage.LoggedInUsersStorage;
+import io.javaclasses.filehub.storage.userStorage.UserId;
+import io.javaclasses.filehub.web.ServerTimeZone;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+import java.time.Period;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+
+@DisplayName("TokenStorage should: ")
+public class LoggedInUsersStorageTest {
+
+    @DisplayName("correctly return existed record by its value.")
```

Comment:
```suggestion
    @DisplayName("correctly return existing record by its value.")
```
Use `existing` if you mean that the record exists at the moment of test.

- [ ] Ok.